pkg/services/postgresql/dbms_arm_template.go

```diff
@@ -57,7 +58,22 @@ var dbmsARMTemplateBytes = []byte(`
 						"startIpAddress": "{{$rule.startIPAddress}}",
 						"endIpAddress": "{{$rule.endIPAddress}}"
 					}
-				}{{if lt $i $count}},{{end}}
+				}{{ if or (lt $i $firewallRulesCount) (gt $virtualNetworkRulesCount -1) }},{{end}}
+				{{end}}	
+				{{range $i, $rule := .virtualNetworkRules}}
+				{
+					"type": "virtualNetworkRules",
+					"apiVersion": "[variables('DBforPostgreSQLapiVersion')]",
+					"dependsOn": [
+						"Microsoft.DBforPostgreSQL/servers/{{ $.serverName }}"
+					],
+					"location": "{{$root.location}}",
+					"name": "{{$rule.name}}",
+					"properties": {
+						"ignoreMissingVnetServiceEndpoint": true,
+        				"virtualNetworkSubnetId": "{{$rule.subnetId}}"
```

Comment:
indent

Reply:
something weird going on with the indentation in VSCode,
After fixing it in `vim` it looks like this in vscode =/

![image](https://user-images.githubusercontent.com/21194782/52964289-d6e73400-3399-11e9-85af-4964115091dc.png)

fixed anyway